network.go

```diff
@@ -1181,7 +1181,8 @@ func (n *network) createEndpoint(name string, options ...EndpointOption) (Endpoi
 	ep.locator = n.getController().clusterHostID()
 	ep.network, err = ep.getNetworkFromStore()
 	if err != nil {
-		return nil, fmt.Errorf("failed to get network during CreateEndpoint: %v", err)
+		logrus.Errorf("failed to get network during CreateEndpoint: %v", err)
```

Comment:
(also left as a comment on the moby PR);

I'm a bit on the fence if we should log this here, given that we're already "handling" the error by returning it. If We want the additional information, perhaps `errors.Wrap(err, "failed to get network during createEndpoint")` (or something along those lines?).

Or perhaps more consistent with the other errors, and actually put this in the exported (`CreateEndpoint`)

Open to suggestions though